context/app/static/js/pages/Workspaces/Workspaces.jsx

```diff
@@ -6,36 +6,15 @@ import { InternalLink } from 'js/shared-styles/Links';
 import WorkspacesAuthenticated from 'js/components/workspaces/WorkspacesAuthenticated';
 import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
 import { StyledDescription } from './style';
-
-function WorkspacesContent() {
-  const { isAuthenticated, isWorkspacesUser } = useAppContext();
-
-  if (!isAuthenticated) {
-    return (
-      <StyledDescription>
-        The workspaces feature is only available if you are logged in and part of the allowed Globus group.{' '}
-        <InternalLink href="/login">Log in</InternalLink> to view saved workspaces or to begin a new workspace.
-      </StyledDescription>
-    );
-  }
-
-  if (!isWorkspacesUser) {
-    return (
-      <StyledDescription>
-        You must be a member of the allowed Globus group to access this feature.{' '}
-        <ContactUsLink variant="body2">Contact us</ContactUsLink> to gain access.
-      </StyledDescription>
-    );
-  }
-
-  return <WorkspacesAuthenticated />;
-}
+import WorkspacesAuthGuard from 'js/components/workspaces/WorkspacesAuthGuard';
 
 function Workspaces() {
   return (
     <>
       <WorkspacesTitle />
-      <WorkspacesContent />
+      <WorkspacesAuthGuard>
+      <WorkspacesAuthenticated />;
```

Comment:
Somewhat odd that Prettier didn't bump the indent here + extra semicolon
```suggestion
        <WorkspacesAuthenticated />
```